### PR TITLE
msgmerge improvements 

### PR DESCRIPTION
**Pull request by fredj**

### CODE REVIEW

**Comment by fredj** [comment]
sort-by-file option is added to avoid this kind of change:
```diff
diff --git a/epfl_authgeoportail/locale/fr/LC_MESSAGES/epfl_authgeoportail-server.po b/epfl_authgeoportail/locale/fr/LC_MESSAGES/epfl_authgeoportail-server.po
index d3a3d75..9805b7b 100644
--- a/epfl_authgeoportail/locale/fr/LC_MESSAGES/epfl_authgeoportail-server.po
+++ b/epfl_authgeoportail/locale/fr/LC_MESSAGES/epfl_authgeoportail-server.po
@@ -1,11 +1,11 @@
 #: epfl_authgeoportail/templates/edit.html:60
-#: epfl_authgeoportail/templates/routing.html:60
 #: epfl_authgeoportail/templates/index.html:72
+#: epfl_authgeoportail/templates/routing.html:60
 msgid "Loading message"
 msgstr ""
 
 #: epfl_authgeoportail/templates/edit.html:74
-#: epfl_authgeoportail/templates/routing.html:74
 #: epfl_authgeoportail/templates/index.html:91
+#: epfl_authgeoportail/templates/routing.html:74
 msgid "search tip message"
 msgstr ""
```

Where only the order changes

**Comment by ochriste** [comment]
+1

File: c2cgeoportal/scaffolds/create/+dot+gitignore_tmpl [hunk 1]
```diff
@@ -1,4 +1,3 @@
-*~
```
Comment (sbrunner) [inline c2cgeoportal/scaffolds/create/+dot+gitignore_tmpl]: Why removing this ?

**Comment by sbrunner** [comment]
Also +1 :-)

File: c2cgeoportal/scaffolds/create/+dot+gitignore_tmpl (hunk 1 above)
Comment (fredj) [inline c2cgeoportal/scaffolds/create/+dot+gitignore_tmpl]: because the backup files from msgmerge are named `xxx.po~`

Comment (sbrunner) [inline c2cgeoportal/scaffolds/create/+dot+gitignore_tmpl]: Then why don't we want to ignore them?

Comment (fredj) [inline c2cgeoportal/scaffolds/create/+dot+gitignore_tmpl]: because the wont be created anymore by using the `backup=none` option

Comment (sbrunner) [inline c2cgeoportal/scaffolds/create/+dot+gitignore_tmpl]: OK, thanks for the full explain :-)

**Pull request merged**